### PR TITLE
[quantization] Use circle_custom.rms_norm in QuantGemma4RMSNorm

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit uses circle_custom.rms_norm in QuantGemma4RMSNorm.

TICO-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: tico/quantization/wrapq/wrappers/gemma4/quant_rmsnorm.py [hunk 1]
```diff
@@ -39,30 +39,41 @@ def __init__(
...
+    def _raw_weight_for_rms_norm(self, x: torch.Tensor) -> torch.Tensor:
+        if self.with_scale:
+            return self.module.weight
+
+        return torch.ones((x.shape[-1],), dtype=x.dtype, device=x.device)
+
+    def _weight_for_rms_norm(self, x: torch.Tensor) -> torch.Tensor:
+        """Return a calibrated or fake-quantized RMSNorm scale tensor."""
+        weight = self._raw_weight_for_rms_norm(x)
+        if self._mode is Mode.QUANT:
```
Comment (shs-park) [inline tico/quantization/wrapq/wrappers/gemma4/quant_rmsnorm.py:61]: `with_scale=False` never collects stats for the synthetic unit `weight`, but quant mode still fake-quantizes it.

A scale-less RMSNorm can quantize the all-ones multiplier from empty observer stats and turn the normalized output into near-zero values instead of preserving unit scale.

```suggestion
        if self._mode is Mode.CALIB:
            self.obs_weight.collect(weight)
        elif self._mode is Mode.QUANT:
            weight = self.obs_weight.fake_quant(weight)
```

**Review by shs-park — COMMENTED** [review]
AI found one or more actionable issues in this PR.
Please check the inline comments when present.

_Reviewed by AI via [oh-my-agents](https://github.com/shs-park/oh-my-agents)_

File: tico/quantization/wrapq/wrappers/gemma4/quant_rmsnorm.py (hunk 1 above)
Comment (mhs4670go) [reply tico/quantization/wrapq/wrappers/gemma4/quant_rmsnorm.py:61]: If `with_scale` is False, `_raw_weight_for_rms_norm` returns `torch.ones`. Therefore, I think it's okay.

**Review by shs-park — APPROVED** [review]
